### PR TITLE
feat: `avm/res/network/private-endpoint` added `groupId` output

### PR DESCRIPTION
**Pull request by segraef**
[avm.res.network.private-endpoint](https://github.com/segraef/bicep-registry-modules/actions/workflows/avm.res.network.private-endpoint.yml)

- Covers https://github.com/Azure/bicep-registry-modules/issues/953
- Added random suffix in e2e tests for KV

image

### CODE REVIEW

**Review by ChrisSidebotham — COMMENTED** [review]
comment above :)

File: avm/res/network/private-endpoint/main.bicep [hunk 1]
```diff
@@ -141,6 +141,9 @@ output name string = privateEndpoint.name
 @description('The location the resource was deployed into.')
 output location string = privateEndpoint.location
 
+@description('The group Id for the private endpoint Group.')
+output groupId string = !empty(privateEndpoint.properties.manualPrivateLinkServiceConnections) ?  privateEndpoint.properties.manualPrivateLinkServiceConnections[0].properties.groupIds[0] : privateEndpoint.properties.privateLinkServiceConnections[0].properties.groupIds[0]
```
Comment (AlexanderSehr) [inline avm/res/network/private-endpoint/main.bicep:145]:
```suggestion
output groupId string = privateEndpoint.properties.?privateLinkServiceConnections[0].properties.groupIds[0] ?? privateEndpoint.properties.?manualPrivateLinkServiceConnections[0].properties.groupIds[0]
```

I 'think' this should work, but please test (:

[the remark this answers is not in the document]
Comment (AlexanderSehr) [reply avm/res/network/private-endpoint/main.bicep:145]: It doesn't @ChrisSidebotham, the module deploys only one PE. 
I guess the `[0]` confuses you, but that simply stems from the fact the the groupIds array acually only allows a single value. Hence in the interface we also ask the user only for a singlar `service`.

**Comment by segraef** [comment]
[avm.res.network.private-endpoint](https://github.com/segraef/bicep-registry-modules/actions/workflows/avm.res.network.private-endpoint.yml) green now

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**